rfcs/imported/0000-dcp-media-types.md

```diff
@@ -0,0 +1,149 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# DCP Media Types
+
+DISCLAIMER: Please note that this RFC has been imported its original Google Doc and went through a design review process prior to the implementation of the current [RFC Process](https://github.com/HumanCellAtlas/dcp-community/blob/master/rfcs/text/0001-rfc-process.md). The original document is [here](https://docs.google.com/document/d/1TqihrgXjct9aDmTJO52_gE2WlpFysB1OkG9C8exmWTw/edit#heading=h.87ix45a71erf). Please be aware that the contents below may potentially be out-of-date as the last-modified date of the original Google Document is October 20th, 2017. 
+
+## Summary
+
+DCP processing involves the transfer and storage of a number of different types of file.  To indicate the type of each file to the DCP, we will use Internet media types [RFC 2046](https://tools.ietf.org/html/rfc2046) during data movement and file storage.
+
+## Author(s)
+
+Original RFC Author:
+
+[Sam Pierson](mailto:spierson@chanzuckerberg.com)
+
+Transcription Authors:
+
+[Arathi Mani](mailto:arathi.mani@chanzuckerberg.com)
+
+[Andrey Kislyuk](mailto:akislyuk@chanzuckerberg.com)
+
+## Shepherd
+
+[Arathi Mani](mailto:arathi.mani@chanzuckerberg.com)
+
+## Motivation
+
+The DCP should not invent new media types; we should append `dcp-type=metadata` or `dcp-type=data` to our Content-Types using the media type "parameter" syntax, e.g. `Content-Type: application/json; dcp-type=”metadata/sample”`. Instead the DCP should strive to use "in-band" media type communication mechanisms where possible.
```

Comment:
This paragraph seems a little mangled.  I think what you are trying to say is "The DCP should not xxx  . Instead it should do yyy".  Right now the Instead clause come after the "should do" clause.

Reply:
Done.